e2e/nomostest/nt.go

```diff
@@ -784,6 +792,34 @@ func (nt *NT) detectGKEAutopilot(skipAutopilot bool) {
 	}
 }
 
+// setupKCC creates the ConfigConnector CR on the KCC cluster.
+// The target project and associated resources (e.g. GSAs) are currently hard
+// coded due to lack of provisioning automation. Eventually we may want to make
+// this more flexible/repeatable for different environments.
+func (nt *NT) setupKCC() {
```

Comment:
suggest avoiding the kcc acronym wherever possible. the acronym has been officially deprecated. tho it's probably ok to leave it in the command flag just so it's short.

Reply:
Use ConfigConnector

Reply:
Done